__test__/unit/blockchain_api/error_scenario_runner.ts

```diff
@@ -24,8 +24,12 @@ const CONTRACT_OWNER = ACCOUNTS[0].address;
 const CREDITOR = ACCOUNTS[1].address;
 const DEBTOR = ACCOUNTS[2].address;
 
-const ZERO_REPAYMENT_AMOUNT = Units.ether(0);
+const ZERO_AMOUNT = Units.ether(0);
 const REPAYMENT_AMOUNT = Units.ether(10);
+const PRINCIPAL_AMOUNT = REPAYMENT_AMOUNT.mul(3);
+// The debtor's balance or allowance must exceed the repayment amount to account
+// for the gas expended in actually sending the repayment.
```

Comment:
Tests are passing, but this is an incorrect diagnosis IMO -- gas is paid in ether, not in ERC20 tokens.  It seems to me that the bigger issue that was solved by these fixes was that the debtor needed to approve the proxy for a transfer allowance.

Please diagnose to the right issue and fix the comments!

Reply:
Added a TODO. Unable to locate source of this error.